Python/Product/PythonTools/PythonTools/LanguageServerClient/LanguageServer.cs

```diff
@@ -62,6 +62,9 @@ public async Task<Connection> ActivateAsync() {
                 return null;
             }
 
+            // Switch to a background thread before starting the process
+            await TaskScheduler.Default;
```

Comment:
Might it have to switch back to the main thread when it's done? Not sure if the caller expects the result on the main thread or not